crates/escrow/src/lib.rs

```diff
@@ -247,6 +257,32 @@ pub mod pallet {
     #[pallet::storage]
     pub type Blocks<T: Config> = StorageMap<_, Blake2_128Concat, T::AccountId, bool, ValueQuery>;
 
+    #[pallet::storage]
+    pub type ThresholdPercentage<T: Config> = StorageValue<_, Percentage, ValueQuery>;
```

Comment:
This will be unset on runtime upgrade, but we don't need it anyway see my comment below.

Reply:
Removed